SpatialGDK/Documentation/content/cross-server-rpcs.md

```diff
@@ -3,11 +3,13 @@
 
 In native-Unreal networking, [RPCs (Unreal documentation)](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/RPCs) are functions which either the client or the server use to send messages to each other over a network connection. 
 
-Cross-server RPCs are a custom solution to take advantage of the SpatialOS distributed server architecture.
+Cross-server RPCs are a custom solution to take advantage of the [zoning]({{urlRoot}}/content/glossary#zoning), which is one of the GDK's options for multiserver development.
```

Comment:
Good spot!!!!

```suggestion
Cross-server RPCs facilitate [zoning]({{urlRoot}}/content/glossary#zoning), which is one of the GDK's options for multiserver development.
```
Think this is probs better like this?

Reply:
Aah, good point - yeah, they facilitate zoning rather than taking advantage of it